src/simplay/src/simplay/components.py

```diff
@@ -121,14 +172,19 @@ def release(self, request: PriorityRequest) -> Release:
 
 class VisualContainer(VisualComponent, Container):
     """
-    Extends the :class:`~simpy.resources.container.Container` class with visualization.
+    Extends the :class:`~simpy.resources.container.Container` class with
+     visualization.
 
     :param env: The environment instance.
     :param id: The id of the component.
     :param capacity: The capacity of the container.
```

Comment:
As this will be the doc the user sees in the end, a user should be able to see whats the type of the param, is that possible? and it should be mentioned that only positive integers are allowed. In javadoc there is the @throws to describe which Exceptions are thrown and in which case: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#throwstag 

Is this documented with this kind of doc or do we need to add that? if yes could you do it with this task or create a new task?

Reply:
The type of param is reflected in the documentation, as can be seen here: https://simplay.readthedocs.io/en/latest/api_reference/simplay.components.html#simplay.components.VisualContainer

I'll add the :raises docstrings which will include why the exception might be raised. From that, users will be able to see that the capacity has to be a positive integer or float.